pkg/reconciler/apiserversource/resources/lavels_test.go

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
@shuheiktgw can you fix the typo in the filename?

Reply:
Sorry about that! I made another PR so would you mind reviewing it? 

https://github.com/knative/eventing/pull/4220